game/scripts/vscripts/components/points/game_length_vote.lua

```diff
@@ -47,4 +47,7 @@ function GameLengthVotes:SetGameLength ()
   end
   DebugPrint ( 'votes ' .. votes.short .. ', ' .. votes.normal .. ', ' .. votes.long .. ' result: ' .. length)
   GameRules.GameLength = length
+  CustomGameEventManager:Send_ServerToAllClients("gamelength_vote_confirmed", {
```

Comment:
i would like to stick to this one, as 
1) it is only needed during pick phase, 
2)nettables do not emit pushes, so it is hard to catch them
3) there is a failover, which in case of pick time reconnect will display "open angel arena"